plugins/inputs/beat/beat_test.go

```diff
@@ -200,7 +201,9 @@ func Test_BeatRequest(test *testing.T) {
 		assert.Equal(test, request.Header.Get("Authorization"), "Basic YWRtaW46UFdE")
 		assert.Equal(test, request.Header.Get("X-Test"), "test-value")
 
-		w.Write(data)
+		if _, err := w.Write(data); err != nil {
```

Comment:
`assert` -> `require` ?

Reply:
We could do this but I've chosen to only convert it if I introduce at least one new `require.NoError` which is not the case here. Please let me know if it's a _must_ to change this...

Reply:
It is not a must, I've just seen an opportunity to replace them. Probably sooner or later we will add `github.com/stretchr/testify/assert` to `imports-blacklist` rule :)